library/Icingadb/Widget/Detail/EventDetail.php

```diff
@@ -319,37 +325,49 @@ protected function assembleDowntimeEvent(DowntimeHistory $downtime)
                     HtmlElement::create('span', ['class' => 'subject'], $this->event->host->display_name)
                 )
             ));
-        $eventInfo[] = new HorizontalKeyValue(t('Entered On'), DateFormatter::formatDateTime($downtime->entry_time));
+        $eventInfo[] = new HorizontalKeyValue(
+            t('Entered On'),
+            DateFormatter::formatDateTime($downtime->entry_time->getTimestamp())
+        );
         $eventInfo[] = new HorizontalKeyValue(t('Author'), [new Icon('user'), $downtime->author]);
             // TODO: The following should be presented in a specific widget (maybe just like the downtime card)
         $eventInfo[] = new HorizontalKeyValue(
             t('Triggered On'),
-            DateFormatter::formatDateTime($downtime->trigger_time)
+            DateFormatter::formatDateTime($downtime->trigger_time->getTimestamp())
         );
         $eventInfo[] = new HorizontalKeyValue(
             t('Scheduled Start'),
-            DateFormatter::formatDateTime($downtime->scheduled_start_time)
+            DateFormatter::formatDateTime($downtime->scheduled_start_time->getTimestamp())
+        );
+        $eventInfo[] = new HorizontalKeyValue(
+            t('Actual Start'),
+            DateFormatter::formatDateTime($downtime->start_time->getTimestamp())
         );
-        $eventInfo[] = new HorizontalKeyValue(t('Actual Start'), DateFormatter::formatDateTime($downtime->start_time));
         $eventInfo[] = new HorizontalKeyValue(
             t('Scheduled End'),
-            DateFormatter::formatDateTime($downtime->scheduled_end_time)
+            DateFormatter::formatDateTime($downtime->scheduled_end_time->getTimestamp())
+        );
+        $eventInfo[] = new HorizontalKeyValue(
+            t('Actual End'),
+            DateFormatter::formatDateTime($downtime->end_time->getTimestamp())
         );
-        $eventInfo[] = new HorizontalKeyValue(t('Actual End'), DateFormatter::formatDateTime($downtime->end_time));
 
         if ($downtime->is_flexible) {
             $eventInfo[] = new HorizontalKeyValue(t('Flexible'), t('Yes'));
             $eventInfo[] = new HorizontalKeyValue(
                 t('Duration'),
-                DateFormatter::formatDuration($downtime->flexible_duration)
+                DateFormatter::formatDuration($downtime->flexible_duration->getTimestamp())
```

Comment:
This doesn't take a timestamp!

Reply:
`getTimestamp()` returns the seconds as expected.